src/s3/http.rs

```diff
@@ -58,6 +55,18 @@ impl Url {
     }
 }
 
+impl Default for Url {
+    fn default() -> Self {
+        Self {
+            https: true,
+            host: String::default(),
```

Comment:
I think this default may crash on display: https://github.com/minio/minio-rs/issues/165